### PR TITLE
Some cleaning

### PR DESCRIPTION
**Pull request by eninja**

### CODE REVIEW

File: spring-petclinic-client/src/scripts/owner-form/owner-form.controller.js [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 angular.module('ownerForm')
     .controller('OwnerFormController', ["$http", '$state', '$stateParams', function ($http, $state, $stateParams) {
-        var self = this;
+        const self = this;
```
Comment (arey) [inline spring-petclinic-client/src/scripts/owner-form/owner-form.controller.js]: ```const``` and ```let``` keywords has been introduced in ES6 (ES2015) and they are compatible with all browsers (ex: IE 10 see https://caniuse.com/#search=let)

**Comment by arey** [comment]
@eninja thank you for your Pull Request. You've done a lot of small changes. For the review, a pull request by changes would be preferable.
I've noticed the Travis build is KO. Could you check why please ?

File: spring-petclinic-client/src/scripts/owner-form/owner-form.controller.js (hunk 1 above)
Comment (eninja) [reply spring-petclinic-client/src/scripts/owner-form/owner-form.controller.js]: they are or they aren't? 
should I revert this patch?

**Comment by eninja** [comment]
It seems like I made some mistakes with var => let || const. 
Hope everything is ok right now.

**Comment by arey** [comment]
Thanks for the JavaScript revert. Could you please check my comment about the required constant?

**Comment by eninja** [comment]
I answered here: https://github.com/spring-petclinic/spring-petclinic-angularjs/pull/23#discussion_r193175937

**Comment by arey** [comment]
@eninja you didn't reply to my last comment:
```
Could you please use a class constant: private static final String REQUIRED = "required";
```

**Comment by arey** [comment]
Thank you for your contribution

**Pull request merged**